src/backends/onnxruntime.c

```diff
@@ -77,32 +77,114 @@ DLDataType RAI_GetDLDataTypeFromORT(ONNXTensorElementDataType dtype) {
   return (DLDataType){ .bits = 0 };
 }
 
-OrtValue* RAI_OrtValueFromTensor(RAI_Tensor* t, RAI_Error *error) {
-  // TODO: create outside and pass?
+// OrtValue* RAI_OrtValueFromTensor(RAI_Tensor* t, RAI_Error *error) {
```

Comment:
should we keep this method commented? will we use it in the future?

Reply:
Deleted